### ISSUE TITLE
Fix Dropout

### ISSUE THREAD

**Issue by Lyubava** [Samsung/veles.znicz#30]
Currently Dropout processes train and validation sets. Need to fix Dropout to process just train set.

**Comment by vmarkovtsev** [comment]
Lyubov created new patchset [1](https://velesnet.ml/gerrit/#/c/170/1)
for change [170](https://velesnet.ml/gerrit/#/c/170) which triggered build [2661](http://velesnet.ml/jenkins/job/VELES_Python_Commit_Znicz/2661/)

**Comment by vmarkovtsev** [comment]
Lyubov created new patchset [2](https://velesnet.ml/gerrit/#/c/170/2)
for change [170](https://velesnet.ml/gerrit/#/c/170) which triggered build [2662](http://velesnet.ml/jenkins/job/VELES_Python_Commit_Znicz/2662/)

**Comment by vmarkovtsev** [comment]
Lyubov created new patchset [1](https://velesnet.ml/gerrit/#/c/217/1)
for change [217](https://velesnet.ml/gerrit/#/c/217) which triggered build [2676](http://velesnet.ml/jenkins/job/VELES_Python_Commit_Znicz/2676/)

**Comment by vmarkovtsev** [comment]
Lyubov created new patchset [2](https://velesnet.ml/gerrit/#/c/217/2)
for change [217](https://velesnet.ml/gerrit/#/c/217) which triggered build [2677](http://velesnet.ml/jenkins/job/VELES_Python_Commit_Znicz/2677/)

**Comment by vmarkovtsev** [comment]
Lyubov created new patchset [1](https://velesnet.ml/gerrit/#/c/222/1)
for change [222](https://velesnet.ml/gerrit/#/c/222) which triggered build [2686](http://velesnet.ml/jenkins/job/VELES_Python_Commit_Znicz/2686/)
